src/semantics/value.ts

```diff
@@ -52,6 +52,7 @@ export enum SemanticsValueKind {
     CLOSURE_CALL,
     CONSTRUCTOR_CALL,
     ANY_CALL,
+    WASM_FUNCTION_CALL,
```

Comment:
We can't introduce such concept in semantic tree, semantic tree should not be coupled with a specific backend.